bin/bossplot

```diff
@@ -72,6 +97,14 @@ def main():
         help = 'Show the subtracted sky flux instead of the object flux.')
     parser.add_argument('--add-sky', action = 'store_true',
         help = 'Add the subtracted sky to the object flux (overrides show-sky).')
+    parser.add_argument('--flux-range', type=str, default='0.5%:99.5%', metavar="MIN:MAX",
```

Comment:
I appreciate that this matches the original behavior but perhaps '0.0:99.5%' is a better default for flux?